test/fuzz/fuzz_doc_symbols.cc

```diff
@@ -0,0 +1,67 @@
+#include "core/Unfreeze.h"
+#include "main/lsp/wrapper.h"
+#include "main/pipeline/pipeline.h"
+#include "payload/payload.h"
+#include "spdlog/sinks/stdout_sinks.h"
+#include "test/helpers/MockFileSystem.h"
+#include "test/helpers/lsp.h"
+#include "test/helpers/position_assertions.h"
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+
+const auto console = spdlog::stdout_logger_mt("console");
+const auto typeErrors = spdlog::stdout_logger_mt("typeErrors");
+const auto rootPath = "/tmp";
+const auto rootUri = fmt::format("file://{}", rootPath);
+
+sorbet::realmain::options::Options mkOpts() {
+    sorbet::realmain::options::Options opts;
+    opts.fs = std::make_shared<sorbet::test::MockFileSystem>(rootPath);
```

Comment:
You'll need to configure `opts` with your chosen rootPath.

https://github.com/sorbet/sorbet/blob/master/main/lsp/wrapper.cc#L65

Reply:
You also need to set `ignoreFlushes` to `true`. There's an ENFORCE in LSPLoop that checks that you did this:

https://github.com/sorbet/sorbet/blob/master/main/lsp/wrapper.cc#L77